src/com/owncloud/android/authentication/AuthenticatorActivity.java

```diff
@@ -1214,74 +1215,77 @@ private void updateServerStatusIconAndText(RemoteOperationResult result) {
         mServerStatusIcon = R.drawable.common_error;    // the most common case in the switch below
 
         switch (result.getCode()) {
-        case OK_SSL:
-            mServerStatusIcon = R.drawable.ic_lock;
-            mServerStatusText = R.string.auth_secure_connection;
-            break;
-
-        case OK_NO_SSL:
-        case OK:
-            if (mHostUrlInput.getText().toString().trim().toLowerCase().startsWith("http://") ) {
-                mServerStatusText = R.string.auth_connection_established;
-                mServerStatusIcon = R.drawable.ic_ok;
-            } else {
-                mServerStatusText = R.string.auth_nossl_plain_ok_title;
+            case OK_SSL:

```

Comment:
I've used the Reformat code option. This is how it looked before the indentation:
```javascript
switch (x) {
case 1: if (something) {
        }
case 2: if (other) {
        }
}
```
In my opinion, this indentation makes the contents of the switch stand out better. :wink: